ts/components/core/logos/LogoPayment.tsx

```diff
@@ -37,8 +38,30 @@ export const IOPaymentLogos = {
   applePay: LogoPaymentApplePay
 } as const;
 
+export const IOPaymentLogosUpperCased: { [key: string]: IOLogoPaymentType } = {
+  MAESTRO: "maestro",
+  MASTERCARD: "mastercard",
+  VISA: "visa",
+  AMEX: "amex",
+  DINERS: "diners",
+  DISCOVER: "discover",
+  JCB: "jcb",
+  UNIONPAY: "unionPay",
+  PAYPAL: "payPal",
+  BANCOMATPAY: "bancomatPay",
+  VPAY: "vPay",
+  PAGOBANCOMAT: "pagoBancomat",
+  SATISPAY: "satispay",
+  POSTEPAY: "postepay",
+  MYBANK: "myBank",
+  APPLEPAY: "applePay"
+} as const;
+
```

Comment:
This is the same ad `IOPaymentLogosCaseMapping`. We should avoid duplicated code for maintainability.

Reply:
If two functions do the same thing and are used for the same purpose,, why should we keep both?  
Don't you think it might make our lives harder to maintain both?